WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -23,41 +21,34 @@
 
 import java.util.List;
 
-public class PeopleManagementActivity extends AppCompatActivity {
-
-    private PeopleAdapter mPeopleAdapter;
+public class PeopleManagementActivity extends AppCompatActivity implements PeopleListFragment.OnPersonSelectedListener {
+    private static final String KEY_PEOPLE_LIST_FRAGMENT = "people-list-fragment";
+    private static final String KEY_PERSON_DETAIL_FRAGMENT = "person-detail-fragment";
```

Comment:
For consistency, let's define these constants with all-caps strings, like earlier in this PR (https://github.com/wordpress-mobile/WordPress-Android/pull/3970/files#diff-76558a95ecd9db2c0dda264a760a7e48R20). Cheers!